.github/workflows/e2e-tests.yml

```diff
@@ -183,6 +185,14 @@ jobs:
         env:
           TERM: xterm
 
+      - name: Upload Replay.io recordings
+        if: failure()
+        uses: replayio/action-upload@v0.4.7
+        with:
+          api-key: ${{ secrets.RECORD_REPLAY_API_KEY }}
+          filter: ${{ 'function($v) { $v.metadata.test.result = "failed" }' }}
```

Comment:
```suggestion
```

Reply:
Let's remove this filter for now. It's helpful to get all of the replays just so we can spot check them.